sms/index.php

```diff
@@ -32,11 +32,20 @@
             try {
                 $phoneNumber = $phoneUtil->parse($smsInput, 'US');
                 $sms = $phoneUtil->format($phoneNumber, PhoneNumberFormat::E164);
+                if (!$phoneUtil->isValidNumber($phoneNumber,'US')){
+                    throw new Exception('Invalid number:'.' '.$phoneNumber);
```

Comment:
It would be better here to either re-use the `NumberParseException`, or to create a new specific Exception subclass (perhaps a subclass of `NumberParseException` with a new `$code` constant).

Reply:
See https://github.com/giggsey/libphonenumber-for-php/blob/master/src/NumberParseException.php as the source of `NumberParseException`.

Reply:
Here's an example of a one-line subclass of `Exception`:
https://github.com/ulsdevteam/php-panopto-upload-api/blob/61a45a5f3a4f573e637b636101a31df494403b8d/lib/Client.php#L15

Reply:
My latest two commits attempt two options for addressing this.  The second is more complex and eliminates the need for the 2nd catch block.